backend/src/user/user.service.ts

```diff
@@ -36,6 +38,85 @@ export class UsersService {
         return result.id as String;
     }
 
+    async getUserById(uid: String) {
+        const user = await this.findUserById(uid);
+        return {
+            id: user.id,
```

Comment:
what if user is null-not found?

Reply:
it throw HttpException in method findUserById